pkg/client/client_test.go

```diff
@@ -467,6 +469,8 @@ func setupTestPlugin(t *testing.T) context.CancelFunc {
 	defer watcher.Close()
 
 	go provider.ServeTestPlugin(debugCtx)
+	// sleep to prevent ParseReattachProviders from reading empty file
+	time.Sleep(time.Millisecond * 50)
```

Comment:
This feels flaky to me, would use a file watcher I think such a check already exists in our tests

Reply:
it seems that error was related to github request limits. Now it works without these lines